Dockerfile

```diff
@@ -18,7 +18,7 @@ COPY --from=elixir-build /build /build
 WORKDIR /build
 ARG MIX_ENV=prod
 ENV MIX_ENV=$MIX_ENV
-RUN mix phx.digest
+RUN RHAI_RUSTLER_FORCE_BUILD=true mix phx.digest
```

Comment:
note to self: we might want to consider making this part of application env in the future